filesystem/flysystem/FlysystemAssetStore.php

```diff
@@ -325,12 +393,12 @@ protected function getOriginalFilename($fileID, &$variant = '') {
 	 *
 	 * The resulting file will look something like my/directory/EA775CB4D4/filename__variant.jpg
 	 *
-	 * @param string $hash
 	 * @param string $filename Name of file
+	 * @param string $hash Hash of original file
 	 * @param string $variant (if given)
 	 * @return string Adaptor specific identifier for this file/version
 	 */
-	protected function getFileID($hash, $filename, $variant = null) {
+	protected function getFileID($filename, $hash, $variant = null) {
```

Comment:
This is the dangerous bit of unsubclassable scalar types - we can't typehint in a way that would catch an error in this refactor